[en]: update ghippo v0.29.0 release notes (#5117)
DaoCloud/DaoCloud-docs: docs/en/docs/ghippo/intro/release-notes.md

This page lists the Release Notes for global management of each version,
so that you can understand the evolution path and feature changes of each version.

## 2024-07-31

### v0.29.0

### Features

- **Added** LDAP server to support LDAPS

> - **Added** installer versions to the platform

Reviewer: ```suggestion
- **Added** installer versions to __Global Management__ -> __Settings__ -> __About__
```
zh version shall be modified too.

### Optimizations

- **Optimized** login and password change functions to support username and password encryption

> - **Optimized** binding resource permissions

Reviewer: ```suggestion
- **Optimized** tip text about binding resources on __Global Management__ -> __Workspace and Folder__ -> __Resource Group__
```
zh text shall be updated too.

### Fixes

- **Fixed** SMTP server settings

> - **Fixed** an issue where OEM configuration could not be installed when set to enable

Reviewer 2: ```suggestion
- **Fixed** an issue where installation fails when OEM is configured `enable`
```

> - **Fixed** the inconsistency issue of English column names in exported reports in __Operations Management__

Reviewer: 英文很好，中文那句不好

## 2024-06-30

### v0.28.0

### Features

- **Added** OAuth2 Identity Provider generic plugin
- **Added** username and userLdapFilter fields to LDAP configuration

> - **Added** installation insight-angent prompt in __Billing Config__ in __Operations Management__

Reviewer 2: ```suggestion
- **Added** prompt for insight-agent installation in __Billing Config__ in __Operations Management__
```

> - **Added** installation insight-angent prompt in __Billing Config__ in __Operations Management__

Reviewer: ```suggestion
- **Added** installation insight-angent prompt in __Global Management__ -> __Operations Management__ -> __Billing Config__
```
zh text shall be updated too.

### Optimizations

> - **Optimized** user creation to automatically remove spaces before and after the username before saving

Reviewer: ```suggestion
- **Optimized** user creation to automatically remove spaces around the username text (if any) before saving
```

> - **Optimized** the binding mechanism of resource search

Reviewer: ```suggestion
- **Optimized** the search mechanism of binding resources
```

> - **Optimized** SDK method `ListWorkspaceUsersByPermission()`

Reviewer: ```suggestion
- **Optimized** a SDK method `ListWorkspaceUsersByPermission()`
```

### Fixes

> - **Fixed** the issue that custom role authorized users can see all workspace

Reviewer: ```suggestion
- **Fixed** an issue that authorized users with a custom role can see all workspaces
```
可以全文一致的 style：`**Fixed** an issue...`

- **Fixed** the issue that the RBAC Access Denied page will appear on the Insight page after closing the browser and reopening it

> - **Fixed** the issue that the report export field name is inconsistent with the page in __Operations Management__

Reviewer: ```suggestion
- **Fixed** an issue that the exported report has inconsistent fields with those shown on the page in __Operations Management__
```
